doc/sailfish-webview-webengine.qdoc

```diff
@@ -0,0 +1,159 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/*!
+  \qmltype WebEngine
+  \inherits Item
+  \inqmlmodule Sailfish.WebEngine
+  \ingroup sailfishos-webview
+  \brief Provides access to the Web engine context
+
+  The \c{WebEngine} component is a singleton which provides access to the Web engine context.
+
+  The \c{WebEngine} QML type is exposed through the \c{Sailfish.WebEngine} QML
+  import namespace.
+
+  Exaple of use:
+
+  \code
+  import QtQuick 2.0
+  import Sailfish.Silica 1.0
+  import Sailfish.WebView 1.0
+  import Sailfish.WebEngine 1.0
+
+  ApplicationWindow {
+      id: root
+      initialPage: Component {
+          Page {
+              id: page
+              WebView {
+                  anchors.fill: parent
+                  active: true
+                  url: "http://www.sailfishos.org"
+              }
+
+              Component.onCompleted: {
+                  WebEngine.addObserver("clipboard:setdata")
+                  WebEngine.onRecvObserve.connect(function(message, data) {
+                      if (message == "clipboard:setdata" && !data.private) {
+                          console.log("Clipboard contents: " + data.data);
+                      }
+                  })
+              }
+          }
+      }
+  }
+  \endcode
+
+  \sa WebVew
+*/
+
+/*!
+  \qmlmethod bool WebEngine::isInitialized()
```

Comment:
We should check if this is really in use... maybe turn into just public C++ method + qml property:

```
Q_PROPERTY(bool initialized READ isInitialized NOTIFY initializedChanged FINAL)
```

Seeing similar glitch in the QMozEngineSettings

Reply:
I can turn this into a property and document it as such as I go.

Reply:
Documented as a property, but I didn't change the code yet (I realise now it's actually in [qtmozembed](https://git.sailfishos.org/mer-core/qtmozembed/blob/0d87492873178eea54af014087499ddb5729a614/src/qmozcontext.h#L36) so I'll create a separate PR).

Reply:
Here's a PR to change `initialized` and `pixelRatio` into properties:

https://git.sailfishos.org/mer-core/qtmozembed/merge_requests/95